docs/platforms/javascript/common/configuration/integrations/localvariables.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: LocalVariables
+description: "Add local variables to exception frames. (default)"
+supported:
+  - javascript.node
+  - javascript.aws-lambda
+  - javascript.azure-functions
+  - javascript.connect
+  - javascript.express
+  - javascript.gcp-functions
+  - javascript.koa
+  - javascript.electron
+---
+
+_Import name: `Sentry.localVariablesSyncIntegration`_
+
+This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
+
+The `onUnhandledRejectionIntegration` registers handlers to capture global unhandled promise rejections.
```

Comment:
Not sure how this line relates to the localVariablesIntegration. Same with the options below.

Reply:
damn, def. a leftover of copying stuff around - will clean up, thanks for noticing!